### PR TITLE
Added Portable directory checkbox

### PR DESCRIPTION
**Pull request by Timodite**
Added an option to toggle Portable/Global Direcotry in Config->Paths

### CODE REVIEW

File: Source/Core/DolphinWX/ConfigMain.cpp [hunk 1]
```diff
@@ -482,6 +492,9 @@ void CConfigMain::InitializeGUITooltips()
 	// Wii - Devices
 	WiiKeyboard->SetToolTip(_("This could cause slow down in Wii Menu and some games."));
 
+	//Paths
+	UsePortable->SetToolTip(_("This disables the global directory and sets the working directory to the executable location, this includes configuration, save data/slots etc. Leave unchecked if unsure. Requires Restart."));
```
Comment (Timodite) [inline Source/Core/DolphinWX/ConfigMain.cpp]: Somebody may recommend a better explanation for this?

Comment (Buddybenj) [inline Source/Core/DolphinWX/ConfigMain.cpp]: Don't capitalize restart at the end.

**Comment by lioncash** [comment]
We have File utility function in Common/FileUtil.h that could simplify much of what you're doing. File::Exists, etc, which would allow you to eliminate inclusion of the stream headers.

**Comment by lioncash** [comment]
Also *please* read the coding style guide. It tells you this in a yellow box before you submit a pull request.

**Comment by delroth** [comment]
If someone checks this box while having already some settings configured, they would lose all their settings for no apparent reason. Really not a fan of the implementation.

**Comment by Timodite** [comment]
@delroth Good point, guess it would be better for overall users not to have this. Closed.

**Comment by Stevoisiak** [comment]
I think a feature like this might be helpful to include as an option in the installer for the next stable build.

**Comment by Timodite** [comment]
@Stevoisiak So maybe a feature where the installer asks whether to include advanced features that a inexperienced user shouldn't change?

**Pull request closed without merging**